app/workers/articles/bust_multiple_caches_worker.rb

```diff
@@ -0,0 +1,14 @@
+module Articles
+  class BustMultipleCachesWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :default, retry: 10
+
+    def perform(article_ids, cache_buster = "CacheBuster")
```

Comment:
Given that we don't use this argument I think we could remove it from the parameters here and skip having to constantize it below.

Reply:
@mstruve you mean forcing it to be always the `CacheBuster` ? if so, I totally agree with it. If needed sometime, this can be injected as property later

Reply:
Yep! So it's just. 
```
cache_buster = CacheBuster 

or 

CacheBuster.bust(article.path)
...
```

Reply:
yeah, good catch! let's do `CacheBuster.bust(article.path)`